torchrl/envs/transforms/transforms.py

```diff
@@ -3893,28 +3893,59 @@ class FrameSkipTransform(Transform):
     Args:
         frame_skip (int, optional): a positive integer representing the number
             of frames during which the same action must be applied.
+        action_interp (bool, optional): whether to perform action interpolation over frame_skip steps.
```

Comment:
```suggestion
        action_interp (bool, optional): whether to perform action interpolation over frame_skip steps.
            Defaults to ``False``.
```

Reply:
Let's add an example that in the docstring
```suggestion
Suggested change
        action_interp (bool, optional): whether to perform action interpolation over frame_skip steps.
            Defaults to ``False``.

        Examples:
            >>> from torchrl.envs import GymEnv
            >>> env = TransformedEnv(GymEnv("CartPole-v1"), FrameSkipTransform(3, action_interp=False))
            >>> print(env.rollout)
            # add print here
            >>> env = TransformedEnv(GymEnv("CartPole-v1"), FrameSkipTransform(3, action_interp=True))
            >>> print(env.rollout)
            # add print here
            
```

Reply:
Suree, will add it in the next commit